vulcan/lib/client/jsx/components/workflow/session/input_group.tsx

```diff
@@ -10,18 +10,19 @@ import {
   sortInputsByLabel
 } from '../../../selectors/workflow_selectors';
 import {InputSpecification} from "../user_interactions/inputs/input_types";
+import {WorkflowContext} from "../../../contexts/workflow_context";
 
 export default function InputGroup({inputs, onChange}: {inputs: InputSpecification[], onChange: (name: string, val: any) => void}) {
   const {state} = useContext(VulcanContext);
-  let {session, workflow, status, data} = state;
+  const workflow = useContext(WorkflowContext);
```

Comment:
the new workflow context allows us to fetch non null workflow.  It will not mount any child component of workflow manager until workflow is non null.  this removes a lot of annoying checks.